apps/storybook/src/Table.stories.tsx

```diff
@@ -102,7 +97,7 @@ export default {
 
 export const Basic: Story<Partial<TableProps>> = (args) => {
```

Comment:
could you remove `args` and controls from all these stories? it hides type errors and is not useful for anyone.

it will also mean less work in the future when we move away from storybook

Reply:
Sounds good, I hope to tackle this in another PR.

Reply:
i mention it now because it will help surface any errors that are getting hidden